boris-http/src/Boris/Http/Api/Build.hs

```diff
@@ -175,6 +176,28 @@ logOf store logs i = do
         DevNull ->
           pure Nothing
 
+logOf' :: Store -> LogService -> BuildId -> EitherT Store.FetchError IO (Maybe [Text])
```

Comment:
Replicating `logOf` but only supporting the DB and only returning `Maybe [Text]` instead of a conduit source. Going this way means not supporting CloudWatch for now?

Reply:
Comments I think address this, I don't think you should serve them differently, but you do need to get the two behaviours together.